fish-rust/src/fds.rs

```diff
@@ -187,3 +187,29 @@ pub fn exec_close(fd: RawFd) {
         }
     }
 }
+
+/// Mark an fd as nonblocking
+pub fn make_fd_nonblocking(fd: &RawFd) -> Result<(), io::Error> {
```

Comment:
moving this from `wutil` to `fds` without moving the C++ version as well can catch future porters offguard. I can live with this one

Reply:
4b4bf541d1 moved the declaration but not the implementation, which I think was a mistake

Reply:
Yeah sloppy of me 😬